### ISSUE TITLE
Not Compatible with Precision Farming Mod

### ISSUE THREAD

**Issue by Aegir2022** [Mogli12/VehicleControlAddon#840]
Hi,
I tested without any other mods,  with VCA the Mod Precision Farming have the Issue that i can not aktivate the ISARIA Scout and ISARIA Pro Aktiv.

**Comment by qqqk2** [comment]
That's right - you need to remap the keys. In the VCA, Alt + B is set to (de)-activate the front hardware by default. It's OK. )
